### ISSUE TITLE
Cannot call a second function after call a first function. (version 8)

### ISSUE THREAD

**Issue by Yothgewalt** [line/line-bot-sdk-go#396]
## Source Code
First function that I called it (Is GetProfile())
And second function that's not working Is replyMessage()
```go
package main
 
import (
	"context"
	"errors"
	"fmt"
	"net/http"
	"os"
	"os/signal"
	"syscall"
	"time"
 
	"github.com/arcmatik/leswitch"
	"github.com/gin-gonic/gin"
	"github.com/line/line-bot-sdk-go/v8/linebot"
	"github.com/line/line-bot-sdk-go/v8/linebot/messaging_api"
	"github.com/line/line-bot-sdk-go/v8/linebot/webhook"
	"github.com/rs/zerolog/log"
)
 
const (
	SERVER_NAME string = "leswitch-events@v1.0.0"
)
 
var (
	ctx = context.Background()
 
	leswitchEventPort = os.Getenv("LWSWITCH_EVENT_PORT")
)
 
func init() {
	time.Local = leswitch.NewTimezone()
	log.Logger = leswitch.NewZerolog()
 
}
 
func main() {
	r := gin.New()
	r.Use(
		leswitch.Logger(SERVER_NAME),
		gin.Recovery(),
	)
 
	v1 := r.Group("/v1")
	{
		bot, err := messaging_api.NewMessagingApiAPI("???")
		if err != nil {
			log.Fatal().Err(err).Msg("Failed to new messaging api client.")
		}
 
		v1.POST("/event/callback", func(ctx *gin.Context) {
			callback, err := webhook.ParseRequest("???", ctx.Request)
			if err != nil {
				switch err {
				case linebot.ErrInvalidSignature:
					ctx.AbortWithError(http.StatusUnauthorized, err)
					return
 
				default:
					ctx.AbortWithError(http.StatusInternalServerError, err)
					return
				}
			}
 
			for _, event := range callback.Events {
				switch e := event.(type) {
				case webhook.MessageEvent:
					switch message := e.Message.(type) {
					case webhook.TextMessageContent:
						switch s := e.Source.(type) {
						case webhook.UserSource:
							profile, err := bot.GetProfile(s.UserId)
							if err != nil {
								ctx.AbortWithError(http.StatusInternalServerError, err)
								return
							}
 
							fmt.Printf("\nDisplay name: %s", profile.DisplayName)
							fmt.Printf("\nPicture url: %s\n", profile.PictureUrl)
 
							if _, err := bot.ReplyMessage(
								&messaging_api.ReplyMessageRequest{
									ReplyToken: e.ReplyToken,
									Messages: []messaging_api.MessageInterface{
										messaging_api.TextMessage{
											Text: profile.UserId,
										},
										messaging_api.TextMessage{
											Text: message.Text,
										},
									},
								},
							); err != nil {
								ctx.AbortWithError(http.StatusInternalServerError, err)
								return
							}
 
						default:
							ctx.AbortWithError(http.StatusInternalServerError, errors.New("Bot cannot use Profile API without User ID."))
							return
						}
					}
 
				default:
					ctx.AbortWithError(http.StatusUnprocessableEntity, errors.New("Unknown event from client."))
					return
				}
			}
		})
	}
 
	for _, route := range r.Routes() {
		log.Info().Str("method", route.Method).Str("route_path", route.Path).Str("handler", route.Handler).Msg("Create a route and exposed")
	}
 
	port := ":4040"
	if leswitchEventPort != "" {
		port = fmt.Sprintf(":%s", leswitchEventPort)
	}
 
	h := &http.Server{
		Addr:    port,
		Handler: r,
	}
 
	go func() {
		if err := h.ListenAndServe(); err != nil && err != http.ErrServerClosed {
			log.Fatal().Err(err).Str("listening_on_port", port).Msg("Failed to start http server.")
		}
	}()
 
	quit := make(chan os.Signal)
 
	signal.Notify(quit, syscall.SIGINT, syscall.SIGTERM)
	<-quit
 
	log.Info().Str("listening_on_port", port).Msg("Shutdown server")
 
	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Second)
	defer cancel()
 
	if err := h.Shutdown(ctx); err != nil {
		log.Fatal().Err(err).Str("listening_on_port", port).Msg("Failed to shutdown server.")
	}
 
	select {
	case <-ctx.Done():
		log.Info().Msg("Timeout of 5 seconds.")
	}
 
	log.Info().Msg("Server exiting...")
}
```

## System Informations

* Go version: 1.21.4
* OS: Mac OS X (Sonoma 14.1.1)

## Expected Behavior

I want to call the both functions for get important data to produce something.

## Current Behavior

Cannot call a second function (such as replyMessage) after call a first function (such as GetProfile)

## Steps to Reproduce

1. Send a message to the bot 
1. Bot retrieve callback from webhook
1. Call GetProfile()(It's working!)
1. Call ReplyMessage() (It's not working!) Why????/

## Logs

<img width="1420" alt="Screenshot 2566-11-19 at 16 41 00" src="https://github.com/line/line-bot-sdk-go/assets/77398391/a7b1b41c-216f-481a-9c46-44fcc63619f6">

**Comment by Yang-33** [comment]
You can get detailed error messages from `ReplyMessageRequestWitHttpInfo`. Please try to check it at first: https://github.com/line/line-bot-sdk-go#how-to-get-response-header-and-error-message

**Comment by Yang-33** [comment]
Hmm... It was reproduced. Request path was changed like `/v2/bot/profile/<USERID>/v2/bot/message/push` on the second request, so the second request failed. We'll check how to fix this. Thank you for your report.

**Comment by Yang-33** [comment]
@Yothgewalt The issue was closed and fixed version was released as `v8.0.2`. Thank you for your report!
